pkg/aws/ec2.go

```diff
@@ -48,6 +48,20 @@ type AssertEC2TagValueEInput struct {
 	InstanceID string
 }
 
+// AssertEC2VolumeTypeInput is used as an input to the AssertEC2VolumeType methods.
+type AssertEC2VolumeTypeInput struct {
+	// The Instance ID that the tag mustbe set on.
+	InstanceID string
+	// The device ID that the volume is mapped to on the instance.
+	DeviceID string
+	// The Volume Type for each volume
+	VolumeType string
+	// The Volume IOPS for each volume
+	VolumeIops int
```

Comment:
**Nit:** Per idiomatic Go conventions `Iops` should be all caps since it's an abbreviation.
```suggestion
	VolumeIOPS int
```